### PR TITLE
Avatar badge : added avatar and badges

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -24,4 +25,58 @@
...
+    height:1rem;
+    width:1rem;
+    background-color: red;
+    font-size:0.75rem;
+    border: 0.1rem solid white;
+}
+
+.badge-button{
+    background-color:blue;
+    color: white;
```
Comment (anuj-kumary) [inline components/badge/badge.css]: Don't hard code color use as a variable

File: components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    
+    <img class="avatar avatar-xsmall" src="https://www.w3schools.com/howto/img_avatar.png" alt="">
+    
+    <img class="avatar avatar-small" src="https://www.w3schools.com/howto/img_avatar.png" alt="">
+
+    <img class="avatar avatar-medium" src="https://www.w3schools.com/howto/img_avatar.png" alt="">
+
+    <img class="avatar avatar-large" src="https://www.w3schools.com/howto/img_avatar.png" alt="">
+
+    <img class="avatar avatar-xlarge" src="https://www.w3schools.com/howto/img_avatar.png" alt="">
```
Comment (anuj-kumary) [inline components/avatar/avatar.html]: Always give alt (alternate) in (avatar, badge).

File: components/badge/badge.css (hunk 1 above)
Comment (MurtazaBagwala07) [reply components/badge/badge.css]: i will change it to variables later

File: components/avatar/avatar.html (hunk 2 above)
Comment (MurtazaBagwala07) [reply components/avatar/avatar.html]: on it , thanks

**Review by anuj-kumary — CHANGES_REQUESTED** [review]
- Don't hard code color define it as a root and use it as variable
- always give some name to alt

File: components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    font-size: 2rem;
+    color:gray;
+    
+}
+
+.badge-number{
+    position: absolute;
+    top: -0.5rem;;
+    right: -0.5rem;
+    display: flex;
```
Comment (rohit8686) [inline components/badge/badge.css:18]: you could move the repetitive code (like flex, height, width, font-size) into separate class and use it

**Review by rohit8686 — APPROVED** [review]

File: components/badge/badge.css (hunk 3 above)
Comment (MurtazaBagwala07) [reply components/badge/badge.css:18]: right , ill try to implement it

**Review by anuj-kumary — APPROVED** [review]
Looks Good!!

**Pull request merged**